.expeditor/config.yml

```diff
@@ -106,6 +70,33 @@ staging_areas:
       workload: pull_request_merged:chef/automate:master:*
 
 subscriptions:
+  # dev -> acceptance promotion
+  # These actions are taken when `/expeditor promote` is run from Slack
+  - workload: project_promoted:{{agent_id}}:dev:*
```

Comment:
Just to confirm:
With the old actions, the channel was the end-state (e.g. only_if == "acceptance"), but now with subscriptions, the channel is the start-state...?

Reply:
Yup